packages/context/src/index.ts

```diff
@@ -3,7 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-export {Binding, BoundValue} from './binding';
+export {Binding, BoundValue, ValueOrPromise} from './binding';
```

Comment:
AFAICT, `ValueOrPromise` is not needed anywhere, can you revert this line please?